### PR TITLE
babili-export

### PR DESCRIPTION
**Pull request by nkutsubos**
**Please check if the PR fulfills these requirements**

- [ ] The commit(s) message(s) follows our [guidelines](https://github.com/talend/tools/blob/master/tools-root-github/CONTRIBUTING.md#commit-message-format) ?
- [ ] Unit tests for the Java changes have been added (for bug fixes / features) ?
- [ ] TUJ for the JavaJet changes have been added (for bug fixes / features) ?
- [ ] Docs have been added / updated (for bug fixes / features) ?
- [ ] The new code does not introduce new technical issues

**What is the current behavior?** (You can also link to an open issue here)

**What is the new behavior?**

**BREAKING CHANGE**

If this PR contains a breaking change, please describe the impact and migration
path for existing applications.
If not please remove this section.

**Other information**:

### CODE REVIEW

File: i18n/plugins/org.talend.repository.hadoopcluster.nl/src/messages_en.properties [hunk 1]
```diff
@@ -263,3 +261,20 @@ DataBricksInfoForm.check.endPoint=EndPoint Label must be specified
...
+HadoopClusterForm.button.useCustomConf.label=Use custom Hadoop configuration
+HadoopClusterForm.button.overrideCustomConf=Set path to custom Hadoop configuration JAR
+HadoopClusterForm.button.overrideCustomConfPath.browse=Select Hadoop configuration file from local file system
+HadoopClusterForm.button.config=Configuration
+HadoopClusterForm.group.customConf=Custom Hadoop configuration
+HadoopClusterForm.webHDFS.encryption=WebHDFS Encryption
+HadoopClusterForm.webHDFS.encryption.useSSLEncryption=Use SSL encryption
+HadoopClusterForm.webHDFS.encryption.useSSLEncryption.trustStorePath=TrustStore path
+HadoopClusterForm.webHDFS.encryption.useSSLEncryption.trustStorePassword=TrustStore password
+HadoopClusterForm.webHDFS.check.trustStorePath=Trust store path must be specified
```
Comment (aparent-talend) [inline i18n/plugins/org.talend.repository.hadoopcluster.nl/src/messages_en.properties]: Trust store should be in one word

File: i18n/plugins/org.talend.repository.hadoopcluster.nl/src/messages_en.properties [hunk 2]
```diff
@@ -263,3 +261,20 @@ DataBricksInfoForm.check.endPoint=EndPoint Label must be specified
...
+HadoopClusterForm.webHDFS.check.trustStorePassword=Trust store password must be specified
```
Comment (aparent-talend) [inline i18n/plugins/org.talend.repository.hadoopcluster.nl/src/messages_en.properties]: same for trust store (one word)

File: i18n/plugins/org.talend.repository.hadoopcluster.nl/src/messages_en.properties [hunk 3]
```diff
@@ -263,3 +261,20 @@ DataBricksInfoForm.check.endPoint=EndPoint Label must be specified
...
+HadoopClusterForm.webHDFS.check.trustStorePassword.invalid=Trust store password is invalid
```
Comment (aparent-talend) [inline i18n/plugins/org.talend.repository.hadoopcluster.nl/src/messages_en.properties]: and same here too

File: i18n/plugins/org.talend.repository.hadoopcluster.nl/src/messages_en.properties (hunk 2 above)
Comment (sbouyer) [reply i18n/plugins/org.talend.repository.hadoopcluster.nl/src/messages_en.properties]: Will fix for next export - Tracking in jira I18N-157. Not a blocker for merging imo.

File: i18n/plugins/org.talend.repository.hadoopcluster.nl/src/messages_en.properties (hunk 1 above)
Comment (sbouyer) [reply i18n/plugins/org.talend.repository.hadoopcluster.nl/src/messages_en.properties]: Will fix for next export - Tracking in jira I18N-157. Not a blocker for merging imo.

File: i18n/plugins/org.talend.repository.hadoopcluster.nl/src/messages_en.properties (hunk 3 above)
Comment (sbouyer) [reply i18n/plugins/org.talend.repository.hadoopcluster.nl/src/messages_en.properties]: Will fix for next export - Tracking in jira I18N-157. Not a blocker for merging imo.

**Pull request merged**